src/components/organisms/events/table/redispatchEventModal.tsx

```diff
@@ -0,0 +1,194 @@
+	const { t } = useTranslation("modals", { keyPrefix: "redispatchEvent" });
+	const { t: tEvents } = useTranslation("events");
+	const { t: tErrors } = useTranslation("errors");
+	const { closeModal } = useModalStore();
+	const { projectsList } = useProjectStore();
+	const [activeDeployment, setActiveDeployment] = useState<string>();
+	const addToast = useToastStore((state) => state.addToast);
+	const [redispatchLoading, setRedispatchLoading] = useState(false);
+	const [eventInfo, setEventInfo] = useState<EnrichedEvent>();
+	const navigate = useNavigate();
+
+	const projectOptions = projectsList.map((project) => ({
+		label: project.name,
+		value: project.id,
+	}));
+
+	const [selectedProject, setSelectedProject] = useState(projectOptions[0]);
+
+	useEffect(() => {
+		const fetchDeployments = async () => {
+			const { data: deployments } = await DeploymentsService.list(selectedProject.value);
+			deployments?.forEach((deployment) => {
+				if (deployment.state === DeploymentStateVariant.active) {
+					setActiveDeployment(deployment.deploymentId);
+				}
+			});
+		};
+		fetchDeployments();
+	}, [selectedProject]);
+
+	const handleRedispatch = useCallback(
+		async () => {
+			if (!activeDeployment) return;
+
+			try {
+				setRedispatchLoading(true);
+				const response = await EventsService.redispatch(eventId, activeDeployment);
+				if (response.error) {
+					throw new Error();
+				}
+				addToast({
+					message: tEvents("table.redispatchSuccess"),
+					type: "success",
+				});
+			} catch {
+				addToast({
+					message: tEvents("table.redispatchFailed"),
+					type: "error",
+				});
+			} finally {
+				setRedispatchLoading(false);
+				closeModal(ModalName.redispatchEvent);
+			}
+		},
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+		[eventId, activeDeployment]
+	);
+
+	const fetchEventInfo = useCallback(async () => {
+		if (!eventId) return;
+		const { data: eventInfoRes, error } = await EventsService.getEnriched(eventId);
+
+		if (error) {
+			addToast({ message: tErrors("errorFetchingEvent"), type: "error" });
+			closeModal(ModalName.redispatchEvent);
+			return;
+		}
+		if (!eventInfoRes) {
+			addToast({ message: tErrors("eventNotFound"), type: "error" });
+			LoggerService.error(namespaces.ui.eventsViewer, tErrors("eventNotFoundExtended", { eventId, error }));
+
+			return;
+		}
+		setEventInfo(eventInfoRes);
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [eventId]);
+
+	useEffect(() => {
+		fetchEventInfo();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [eventId]);
+
+	const handleNavigation = () => {
+		if (!eventInfo) return;
+
+		const { destinationId, destinationType, projectId } = eventInfo;
+		const path = destinationType === EventTypes.trigger ? "triggers" : "connections";
+
+		navigate(`/projects/${projectId}/${path}/${destinationId}/edit`);
+	};
+
+	return (
+		<Modal className="w-700" hideCloseButton name={ModalName.redispatchEvent}>
```

Comment:
Let's try to invert the color scheme of the modal from light to dark.
Right now the modal is bigger than most of the modals in our system, and when it opens, we're getting too much white on a dark screen